om-2.0.rdf

```diff
@@ -31592,6 +31603,13 @@
     <om:symbol>pH</om:symbol>
   </owl:Class>
 
+    <!-- pH Level Ontology -->
+
+  <owl:Class rdf:about="&om;pHLevel">
+    <rdfs:label xml:lang="en">pH level</rdfs:label>
+    <rdfs:subClassOf rdf:resource="&om;Acidity"/>
+  </owl:Class>

```

Comment:
Would you be able to explain the reasoning behind this? Why do we need "pH level"? Does the `Acidity` class not do the job?

Reply:
I don't believe that is sufficient justification for a subclass. How about adding an `om:alternativeLabel` of "pH level" to `om:Acidity`?

Reply:
Okay, that sounds good. I have added the alternative label to om:Acidity.